services/distribution/src/test/resources/dgc/cclrules.json

```diff
@@ -0,0 +1,26 @@
+[
+  {
+    "identifier": "CCL-DE-0001",
+    "version": "1.0.0",
+    "country": "DE",
+    "hash": "7221d518570fe9f4417c482ff0d2582a7b6440f243a9034f812e0d71611b611f"
+  },
+  {
+    "identifier": "CCL-DE-0002",
+    "version": "1.0.2",
+    "country": "DE",
+    "hash": "7021d518570fe9f4417c482ff0d2582a7b6440f243a9034f812e0d71611b611l"
+  },
+  {
+    "identifier": "CCL-DE-0001",
+    "version": "1.0.3",
+    "country": "DE",
+    "hash": "6821d518570fe9f4417c482ff0d2582a7b6440f243a9034f812e0d71611b611f"
+  },
+  {
+    "identifier": "CCL-DE-0001",
+    "version": "1.0.3",
```

Comment:
same version AND same identifier !!!??? that's not defined...

Reply:
This is just for testing purposes. The hashes are different and just 1 of them would have a file in the resources.